### PR TITLE
Update `clean-css` To `v5.3`

### PR DESCRIPTION
**Pull request by Casey-Kiewit**
Updates `clean-css` to the latest version. Updates a test to reflect the functionally identical but different output the latest version of `clean-css` produces.

Resolves https://github.com/terser/html-minifier-terser/issues/149.

### CODE REVIEW

**Comment by RamyaKVelivala** [comment]
When are you planning to merge this pull request, we are unable to use "html-minimizer-webpack-plugin" because of transitive dependency of "clean-css" using older version.

**Comment by DanielRuf** [comment]
@sibiraj-s can you handle the release of a new version? That would be great.

@RamyaKVelivala for a short-term solution please take a look at yarn resolutions and npm overrides:
https://classic.yarnpkg.com/en/docs/selective-version-resolutions/
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides

File: package.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
   },
   "dependencies": {
     "camel-case": "^4.1.2",
-    "clean-css": "5.2.0",
+    "clean-css": "~5.3.2",
```
Comment (XhmikosR) [inline package.json:78]: You really should have used `^` here so that when 5.4.x etc is out no one needs to change the package here.

Comment (XhmikosR) [reply package.json:78]: Well, if they offer to update the deps in a timely manner, sure. Otherwise, better use `^` and when tests break, someone updates the fixtures. That was the consumers won't have to wait for a new html-minifier-terser release.

Both solutions are a compromise, of course; it depends from which point of view one sees the issue. :)

Comment (sibiraj-s) [reply package.json:78]: Yes, I didn’t want to use `~` or `^` in case of clean-css, just pin to a fixed version. since they introduced a breaking change in semver minor. But if it a one time thing like an accident or a bug fix actually. May be we can ignore this once.

As this might allow less maintenance releases from our end, also allows users to stay upto date on security patches.

I am fine on both options have a caret or pin it. But if clean-css doesn’t follow semver or this happened more than once, I recommend pin it.

Comment (XhmikosR) [reply package.json:78]: Yeah, semver is hard to follow, I've been there myself 😛

Thanks for considering!
